### ISSUE TITLE
Multiplatform version

### ISSUE THREAD

**Issue by rjaros** [lucapiccinelli/konad#2]
It looks like a pure Kotlin library. It would be great if it was a multiplatform project.

**Comment by lucapiccinelli** [comment]
Yes you are right. Onestly I don't have experience on Kotlin compilers different from those on JVM... but it was already on my mind. I would like to use Konad as a playground to learn about other compilers.

But also contributions are welcome 😁
